noteDB.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE IF NOT EXISTS 'notesDB';
+
+CREATE TABLE IF NOT EXISTS 'notesTB' (
+    title VARCHAR(30) NOT NULL,
+    id SMALLINT NOT NULL AUTO_INCREMENT,
+    modifieddate TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    note TEXT DEFAULT NULL,
```

Comment:
i think it is good using TEXT on the note, if you want the length of note unlimited.
Is it necessary  to set the NULL on your TEXT col?

Reply:
why the default is NULL instead ""(empty string)

Reply:
Apparently TEXT is old and should not be applied to new code.
VARCHAR(MAX) should be the better option here, fixed this!
No it is not necessary but I felt you should be able to make a note with only a title and empty text.

Reply:
According to Caché Development Guides
https://cedocs.intersystems.com/latest/csp/docbook/DocBook.UI.Page.cls?KEY=GSQL_langelements_null

<q>The SQL zero-length string is not recommended for use as a field input value or a field default value. Use NULL to represent the absence of a data value.</q>